### ISSUE TITLE
to do: algorithms, flowcharts and powerpoint

### ISSUE THREAD

**Issue by fishtez** [fishtez/blackjack#18]

**Comment by fishtez** [comment]
i've uploaded flowcharts for wager, loadSave and newSave. i'll make one for main, too. Zubeir are you still having trouble downloading visio?

**Comment by Zubz21** [comment]
Yeah but im going to try my dads computer. Hopefully its just my computer

**Comment by fishtez** [comment]
flowchart for main is posted to report file

**Comment by fishtez** [comment]
i think the only flowchart we still need is for Play. we still need algorithms for checkAce and checkAceDealer

**Comment by fishtez** [comment]
i think we just need the flowchart for play added to the report on googledocs

**Comment by DavidL47** [comment]
yea i think so i too just redid checkAce flow chart so it matches the
changes i made

**Comment by DavidL47** [comment]
for the presentation im probally only going to make one slide for both
checkdealer ace and checkace since they are practically the exact same code
except it passes in dealer stuff instead of player

**Comment by fishtez** [comment]
makes sense

**Comment by fishtez** [comment]
is there anything else you guys want to add to the powerpoint before i upload it?

**Comment by Zubz21** [comment]
Are you bringing the USB Marnie. I would bring my laptop but it has a minor
virus

**Comment by fishtez** [comment]
I have it on usb
